operations/helm/charts/mimir-distributed/values.yaml

```diff
@@ -1151,8 +1156,151 @@ querier:
   env: []
   extraEnvFrom: []
 
+
+ruler_querier:
+  replicas: 10
```

Comment:
10 replicas seems a bit high for default, considering the user path querier comes with 2. Maybe we default to 2 here as well?

Reply:
@Logiraptor  agreed thx.
@dimitarvdimitrov It should be off by default.  I can update the small and large values to include the service. 

```
ruler_querier_service:
  enabled: false
```